test/integration/features/basic.feature

```diff
@@ -84,6 +84,58 @@ Feature: Basic
       172.30.1.1:5000
       """
 
+  Scenario: Deploy ruby-ex s2i image
```

Comment:
That said, I think we can spit this ling scenario up into several smaller ones:
* User can login to OpenShift
* User can create new project
* User can deploy the example Ruby application ruby-ex
* User can delete project

Having multiple scenarios will also help with the "When", "Then", "And" logic